### PR TITLE
MRG: use get_fdata in docs

### PR DESCRIPTION
**Pull request by matthew-brett**
We will soonish deprecate get_data, so remove from the docs now, so as
not to trip up new users.

### CODE REVIEW

File: doc/source/nibabel_images.rst [hunk 1]
```diff
@@ -220,21 +220,38 @@ False
...
+
+>>> farray_img = nib.Nifti1Image(image_data.astype(np.float64), affine)
+>>> farray_data = farray_img.get_fdata()
+>>> farray_data.dtype == np.dtype(np.float64)
+True
+
+There was no cast, so the array returned is exactly the array attached to the image:
+
+>>> image_data is array_img.dataobj
+False
```
Comment (effigies) [inline doc/source/nibabel_images.rst]: I might be misreading, but from the above sentence, I'd think this ought to be:

```Python
>>> farray_data is farray_img.dataobj
True
```

Comment (matthew-brett) [reply doc/source/nibabel_images.rst]: Good spot - you're right, but then this test fails, revealing the bug I think I have fixed in https://github.com/nipy/nibabel/pull/638

**Comment by matthew-brett** [comment]
This PR needs to be merged after https://github.com/nipy/nibabel/pull/638 (with fixes suggested above).

**Comment by effigies** [comment]
These stochastic AppVeyor failures seem to be cropping up again. Are you okay if I just go ahead and merge?

**Comment by matthew-brett** [comment]
Yes, go for it ...

**Pull request merged**